src/url-params/url-params.js

```diff
@@ -188,14 +233,14 @@
                 urlManager.updateUrl();
 
                 // Get tags that need to be activated, based on current URL
-                var tagsToShow = urlManager.getTagsForCurrentState();
+                var hotspotTagsToClick = urlManager.getTagsForCurrentState();
```

Comment:
these are slugs right? i should have mentioned this. can this say something like hotpoststoclickslugs or something?